index.html

```diff
@@ -114,6 +114,36 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
```

Comment:
Make sure to add a comment on top of your card. 
The top comment from the template 
`<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->` 
should be changed to `<!-- ________ your_name card START ________  -->`